src/main/java/it/gov/pagopa/pu/debtpositions/service/BaseDebtPositionOperationService.java

```diff
@@ -75,7 +77,11 @@ public WorkflowCreatedDTO execute(DebtPositionDTO debtPositionDTO, List<Installm
     if(!OrganizationStatus.ACTIVE.equals(org.getStatus())){
       throw new InvalidValueException("Provided organization is not ACTIVE");
     }
-    authorizeOperatorOnDebtPositionTypeService.authorize(org.getIpaCode(), debtPositionDTO.getDebtPositionTypeOrgId(), operatorExternalUserId);
+    DebtPositionTypeOrg debtPositionTypeOrg = authorizeOperatorOnDebtPositionTypeService.authorize(org.getIpaCode(), debtPositionDTO.getDebtPositionTypeOrgId(), operatorExternalUserId);
+
+    if (!isDisableDebtPositionTypeOrgAllowed(debtPositionTypeOrg)) {
```

Comment:
this method should not accept any parameter, the check on flagActive will be always the same
```suggestion
    if (!isDisableDebtPositionTypeOrgAllowed() && !debtPositionTypeOrg.isFlagActive()) {
```